src/pseudopeople/interface.py

```diff
@@ -126,6 +155,8 @@ def _generate_dataset(
         import dask
         import dask.dataframe as dd
```

Comment:
@zmbc Remind me why you prefer having dask imports in local scope? It leads to a weird thing where we `import dask.dataframe as dd` if typechecking and then import it here during runtime if the engine is dask.